### PR TITLE
Kafka topic addition in Env file as part of JIRA key-> SB-25750

### PR DESCRIPTION
**Pull request by Hari-1027**
Added a new Kafka Topic in sunbird_program.env file as part of QUML bulk upload JIRA key-> SB-25750

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [Yes ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

No testing was handled. Below Kafka Topic in sunbird_program.env file as part of QUML bulk upload JIRA key-> SB-25750

sunbird_question_bulkupload_topic

### Checklist:

- [Yes ] My code follows the style guidelines of this project
- [Yes ] I have performed a self-review of my own code
- [Yes ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [Yes ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [Yes ] New and existing unit tests pass locally with my changes
- [Yes ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_program.env [hunk 1]
```diff
@@ -16,6 +16,9 @@ telemetry_service_endpoint=v1/telemetry
 telemetry_service_host=http://telemetry-service:9001
 sunbird_kafka_host={{groups['sunbird-processing-cluster-kafka']|join(':9092,')}}:9092
 sunbird_auto_creation_topic={{ sunbird_auto_creation_topic }}
+sunbird_question_bulkupload_topic={{ env_name }}.av.question.bulkupload
+sunbird_kafka_bulkupload_consumer_group_id={{ env_name }}.av.qstn.bulkupload
+sunbird_assessment_service_base_url={{ sunbird_kp_assessment_service_base_url }}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_program.env:21]: Please change this to `http://assessment-service:9000`

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Please check comments

File: ansible/roles/stack-sunbird/templates/sunbird_program.env (hunk 1 above)
Comment (Hari-1027) [reply ansible/roles/stack-sunbird/templates/sunbird_program.env]: @keshavprasadms changed the value as suggested and updated the PR.

Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/templates/sunbird_program.env]: @Hari-1027 Merging this. Please note the value does not end with `/`. Your need to append it in your code.
Or else you can update the variable value to end with a `/`

Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/templates/sunbird_program.env]: For reference - https://github.com/project-sunbird/sunbird-devops/blob/3d3335348ba61b1d4e33bb5cad2658ef8156d30e/ansible/roles/stack-sunbird/templates/sunbird_program.env#L23

**Review by keshavprasadms — APPROVED** [review]

File: ansible/roles/stack-sunbird/templates/sunbird_program.env (hunk 1 above)
Comment (Hari-1027) [reply ansible/roles/stack-sunbird/templates/sunbird_program.env]: Thank you @keshavprasadms will make sure to append the '/' in our code.

**Pull request merged**